### PR TITLE
Give suggestions on both error and success after fetching LATEST

### PR DESCRIPTION
**Pull request by andralex**

### CODE REVIEW

File: posix.mak [hunk 1]
```diff
@@ -64,9 +64,9 @@ LATEST:=$(shell cd ${DMD_DIR} && \
   git tag | grep '^v[0-9][0-9.]*$$' | sed 's/^v//' | sort -nr | head -n 1)
 endif
 ifeq (,${LATEST})
-  $(error Could not fetch latest version)
+  $(error Could not fetch latest version, place LATEST=2.xyz in the command line)
```
Comment (CyberShadow) [inline posix.mak]: `2.xyz.t` ?

**Comment by CyberShadow** [comment]
>  per title 

The pull request title is not visible when browsing commits.

File: posix.mak (hunk 1 above)
Comment (andralex) [inline posix.mak]: It's either one or the other. Which is more common?

**Comment by andralex** [comment]
@CyberShadow fixed

File: posix.mak (hunk 1 above)
Comment (CyberShadow) [inline posix.mak]: Um? All recent releases are in the form `2.NNN.N`. Not sure what you mean.

**Pull request merged**